opencog/atoms/atom_types/atom_types.script

```diff
@@ -32,6 +32,7 @@ INDEFINITE_TRUTH_VALUE     <- TRUTH_VALUE
 FUZZY_TRUTH_VALUE          <- TRUTH_VALUE
 PROBABILISTIC_TRUTH_VALUE  <- TRUTH_VALUE
 EVIDENCE_COUNT_TRUTH_VALUE <- TRUTH_VALUE
+TTRUTH_VALUE               <- TRUTH_VALUE "TTruthValue"
```

Comment:
I suggest name it `TORCH_TRUTH_VALUE ` as `TTRUTH_VALUE` is unclear and ambiguous - usually `T` at the beginning sounds as `Type`.  And rename `TTruthValue` to `TorchTruthValue` as well for clarity.